lib/APIPlugin.js

```diff
@@ -37,17 +37,11 @@ class APIPlugin {
 
 			params.normalModuleFactory.plugin("parser", parser => {
 				Object.keys(REPLACEMENTS).forEach(key => {
-					parser.plugin(`expression ${key}`, expr => {
-						const dep = new ConstDependency(REPLACEMENTS[key], expr.range);
-						dep.loc = expr.loc;
-						parser.state.current.addDependency(dep);
-						return true;
-					});
-
+					parser.plugin(`expression ${key}`, ParserHelpers.toConstantDependency(REPLACEMENTS[key]));
 					parser.plugin(`evaluate typeof ${key}`, ParserHelpers.evaluateToString(REPLACEMENT_TYPES[key]));
 				});
 				IGNORES.forEach(key => {
-					parser.plugin(key, () => true);
+					parser.plugin(key, ParserHelpers.returnTrue);
```

Comment:
I probably went a bit too far here 😨 
All those lonely `() => true`, i thought i give them a new home.
(at least it only creates one function instead of many. but thats the best excuse I can come up with.)
that being said, happy to revert :)